CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Add a new node role 'search' which is dedicated to provide search capability ([#4689](https://github.com/opensearch-project/OpenSearch/pull/4689))
 - Introduce experimental searchable snapshot API ([#4680](https://github.com/opensearch-project/OpenSearch/pull/4680))
 - Recommissioning of zone. REST layer support. ([#4624](https://github.com/opensearch-project/OpenSearch/pull/4604))
+- Make the index restored by remote snapshot as read only but allow deletion ([#4764](https://github.com/opensearch-project/OpenSearch/pull/4764))
```

Comment:
"index restored by remote snapshot" is an accurate of normal snapshot restores as well so this should probably be more descriptive. Something like "Make searchable snapshot indexes read-only but allow deletion".

Reply:
Updated PR description